LibraryOfMedicine/WithOrWithoutPrescription.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace LibraryOfMedicine
+{
+    public enum dispensation
+    {
+        withPrescription,
+        withoutPrescription
+    }
+}
```

Comment:
Стиль! Имена перечислений и их членов принято писать с большой буквы
```suggestion
namespace LibraryOfMedicine
{
    public enum Dispensation
    {
       WithPrescription,
        WithoutPrescription
    }
}
```